### ISSUE TITLE
User with disabled password is not working

### ISSUE THREAD

**Issue by phajduk** [frankfarrell/terraform-provider-redshift#22]
In README there is following example:
```
resource "redshift_user" "testuser"{
  "username" = "testusernew",
  "password_disabled" = true
  "connection_limit" = "1"
}
```
this is not working from multiple reasons:
- [password field is required](https://github.com/frankfarrell/terraform-provider-redshift/blob/master/redshift/resource_redshift_user.go#L30) so validation fails
- create user query is broken according to the [Redshift documentation](https://docs.aws.amazon.com/redshift/latest/dg/r_CREATE_USER.html#r_CREATE_USER-examples)

I'm going to pare pull request for this issue.

**Comment by phajduk** [comment]
PR: https://github.com/frankfarrell/terraform-provider-redshift/pull/23

**Comment by frankfarrell** [comment]
Closed by #23
